docker/termui/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.17.6 as builder
+RUN apt-get update && apt-get install -y
+WORKDIR /go/mx-chain-go
+COPY . .
+WORKDIR /go/mx-chain-go/cmd/termui
+RUN go build -i -v
+RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/mx-chain-go/go.mod | grep arwen-wasm-vm | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so || :
```

Comment:
change to multiversx location ?

Reply:
ah, no rebranding in master yet

Reply:
will do it in a follow up merge PR from `master` -> `rc/v1.4.0`